docs/dev/release.rst

```diff
@@ -47,12 +47,19 @@ In particular, **don't** prefix the tag with ``v``.
 
 The `ci.yaml`_ GitHub Actions workflow uploads the new release to Docker Hub.
 
+3. Create a GitHub release
+--------------------------
+
+Add a new GitHub release for this version.
+The release title should be the same as the version number.
+The description of the release should be the ``CHANGELOG.rst`` entry for this release, converted to Markdown.
```

Comment:
Can't wait to see you migrate this to Markdown 😆

Reply:
Maybe I should do that now and get it over with, since I'm in here. :)